monitor/processor.go

```diff
@@ -18,6 +18,10 @@ import (
 	"github.com/golang/protobuf/proto"
 )
 
+const (
+	maxPunishRetry = 5
```

Comment:
Add todo or ticket to move this to config.

Reply:
I do not it is necessary to make this configurable

Reply:
how long does it wait between each retry?

Reply:
It depends on the number of validator and duration of a puller. i.e. (number of validator) * (duration of a puller)